components/server/src/auth/resource-access.ts

```diff
@@ -209,6 +218,8 @@ export class OwnerResourceGuard implements ResourceAccessGuard {
                 }
             case "workspaceLog":
                 return resource.subject.ownerId === this.userId;
+            case "prebuild":
+                return resource.workspace.ownerId === this.userId;
```

Comment:
Shouldn't this be accessible by the team too? (E.g. `resource.members.some(...)` as above)

Reply:
Ah sorry, I see that this is handled via the `RepositoryAccessGuard` below. Makes sense. 💡

Reply:
Hm. No, you are right! It has the same effect (because on the happy path, if you are member of a team, you are likely to have read access to the repo.
I felt into the cargo-cult trap: We should have the same for workspaceLog. Will fix. :+1:

Reply:
Fixed in https://github.com/gitpod-io/gitpod/pull/10940/commits/0a37cf9f329b23d168d11fda8c80af479840ae0f